docs/versioning/how-to-view-versions.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+nav_order: 5
+title: How to view versions  
+has_children: false
+parent: Versioning
+---
+# How to view versions
+
+
+## Viewing version info in the Registry
+
+
+### Step 1. Navigate to Versioning tab in the item record in the Registry
+
+- Ensure you're logged in.
+- Navigate to the Registry.
+- Navigate to the item record.
+- Click on the "VERSIONING" tab as shown below.
```

Comment:
Disclaimer here that this may not appear for non ENTITY types

Reply:
hmm, maybe we need to hyperlink to the overview section with that list.

Reply:
added faq entry